tests/integration/test_eoc_journal.py

```diff
@@ -115,6 +163,52 @@ def test_pb_answer_blocks_listed_in_edit_view(self):
             label = item.find_element_by_css_selector('label')
             self.assertEqual(label.text, title)
 
+    def test_pb_answers_listed_in_student_view(self):
+        selected_block_ids = default_pb_answer_block_ids[1:]
+        self.configure_block(selected_pb_answer_blocks=selected_block_ids)
+
+        element = self.get_element_for_selected_pb_answers()
+
+        items = element.find_elements_by_css_selector('.eoc-selected-answers-section')
+        self.assertEqual(len(items), 1)
+        
+        section_name = items[0].find_elements_by_css_selector('h2')[0]
+        self.assertEqual(section_name.text, 'Second Section')
+
+        answers_in_section = items[0].find_elements_by_css_selector('.eoc-selected-answer')
+        self.assertEqual(len(answers_in_section), 2)
+      
+        expected_answers_data = [
+            FakeAnswer(None, 'Not answered yet.', 'Hello, who are you?')
+        ]
+
+        expected_answers_data += default_answers_data
+
+        for element, expected in zip(answers_in_section, expected_answers_data):
+            question = element.find_elements_by_css_selector('h3')[0]
+            answer = element.find_elements_by_css_selector('p')[0]
```

Comment:
Same here: user `find_element_by_css_selector(...)` instead of `find_elements_by_css_selector(...)[0]` when only interested in the first matching element.